src/librustc/infer/error_reporting/mod.rs

```diff
@@ -847,7 +847,7 @@ impl<'a, 'gcx, 'tcx> InferCtxt<'a, 'gcx, 'tcx> {
                                                "{} may not live long enough",
                                                labeled_user_string);
                 err.help(&format!("consider adding an explicit lifetime \
-                                   bound `{}: 'static`...",
+                                   bound `{}: 'static + `...",
```

Comment:
Yeah, I don't know what's best. I'm not sure there's an *easy* fix to improve these suggestions, other than maybe using the word "where clause" instead of "binding". I think I'm reluctant to include the trailing `+` by itself.